internal/status/route_status.go

```diff
@@ -301,7 +299,7 @@ func updateRouteObject(mRoute *routev1.Route, updateOption UpdateOptions, retryN
 
 	// If we find a hostname in the present update, let's first remove it from the existing status.
 	for i := len(mRoute.Status.Ingress) - 1; i >= 0; i-- {
-		if utils.HasElem(hostnames, mRoute.Status.Ingress[i].Host) {
+		if utils.HasElem(hostnames, mRoute.Status.Ingress[i].Host) && mRoute.Status.Ingress[i].RouterName == lib.AKOUser {
```

Comment:
@sudswasavi @monotosh-avi : Will it be good to add "RouterName" check in getRoutes() function before "routes" getting added to routemap? I think this will save few cycles.

Reply:
As discussed, we would need to fetch all routes for now.